### DISCUSSION TITLE
Anesthesia & fMRI

### DISCUSSION THREAD

**Discussion by pcklink** [PRIME-RE/prime-re.github.io#71] [General]
What are your current opinions on the best anesthetic to get sensory (visual) evoked BOLD signals in anesthetized macaque fMRI? 

The [Basso et al. paper](https://doi.org/10.1016/j.neuroimage.2020.117667) seems to lean a bit towards propofol, but others also use ketamine/medetomidine. I guess iso/sevoflurane would suppress the activity too much, so propofol and ketamine seem to be the prime candidates, but perhaps there are other opinions out there?

We use ketamine/medetomidine for occasional anesthetized anatomical scans but I have no experience with anesthetized functional scans. Would appreciated your input!

**Comment by afni-dglen** [comment]
I can't comment on the type of anesthesia, but I would say, as usual, to be careful regarding artifacts from the use of the ventilation and monitoring equipment. In previous studies, we found the ventilators or the blood pressure measurement from the cuff can impose specific frequencies and periodic signals. The effect may either an "apparent motion" or the imposed oxygenation of the blood.  When there is little else in the signal, these artifacts become the most important part of the signal. Handling the specific frequencies or censoring specific TRs can mitigate some of these issues, but one has to be aware of these problems in the first place. Careful quality control that looks at global and regional/radial correlation can help.

https://www.frontiersin.org/articles/10.3389/fneur.2021.659002/full
https://www.sciencedirect.com/science/article/pii/S1053811921002731?via%3Dihub

**Comment by niksirbi** [comment]
I cannot speak for macaques, but at the German Primate Centre we had good results in **marmosets** with the combination of **(dex)medetomidine via continuous IV infusion** + **isoflurane supplied via mask** (low concentration 0.4-0.6%). The animals are self-breathing (no intubation needed), and it's quite safe against anesthesiological complications, for example:
- if sedation is insufficient (e.g. fast or irregular breathing) -> increase the isoflurane concentration
- if breathing becomes too slow -> decrease the isoflurane concentration
- if there is bradycardia or bradyarrhythmias (a possible side-effect of medetomidine) -> supply the ultra-specific antagonist atipamezole

As usual, good cardiorespiratory monitoring is required for this to work as intended.
Our visually-evoked fMRI activations maps looked great with this approach. A preprint about this should be coming *soonish*.

For context, the (dex)medetomidine + isoflurane combo is the current consensus protocol for rsfMRI in [mice](https://www.sciencedirect.com/science/article/pii/S1053811919308699) and [rats](https://www.nature.com/articles/s41593-023-01286-8). That's where we were inspired from.

> **Warning**
> I'm not aware of any publication using this protocol in macaques, and I also don't know what the right doses would be in that case.

**Comment by glasserm** [comment]
We published this protocol in 2020: https://www.sciencedirect.com/science/article/pii/S1053811920302871 in macaques.

**Comment by PRIME-DRE** [comment]
Anesthesia with dexmedetomidine + low-dose isoflurane was also effectively used
in marmoset
https://www.sciencedirect.com/science/article/pii/S1053811922000945?via%3Dihub
and in night monkey
https://link.springer.com/article/10.1007/s00429-022-02591-x


________________________________
差出人: 'glasserm' via NHP-MRI ***@***.***>
送信日時: 2023年9月21日 22:43
宛先: PRIME-RE/prime-re.github.io ***@***.***>
CC: Subscribed ***@***.***>
件名: Re: [PRIME-RE/prime-re.github.io] Anesthesia & fMRI (Discussion #71)


We published this protocol in 2020: https://www.sciencedirect.com/science/article/pii/S1053811920302871 in macaques.

—
Reply to this email directly, view it on GitHub<https://github.com/PRIME-RE/prime-re.github.io/discussions/71#discussioncomment-7070763>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BBSQQAPQDVMQUYXHKLWXTK3X3Q77BANCNFSM542OYVXQ>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

--
You received this message because you are subscribed to the Google Groups "NHP-MRI" group.
To unsubscribe from this group and stop receiving emails from it, send an email to ***@***.******@***.***>.
To view this discussion on the web visit https://groups.google.com/d/msgid/nhp-mri/PRIME-RE/prime-re.github.io/repo-discussions/71/comments/7070763%40github.com<https://groups.google.com/d/msgid/nhp-mri/PRIME-RE/prime-re.github.io/repo-discussions/71/comments/7070763%40github.com?utm_medium=email&utm_source=footer>.

**Comment by pcklink** [comment]
And what do people use to keep the eyes open and hydrated when doing anaesthetized visual stimulation in the magnet? We're not after anything precise, but are looking to verify that we can measure a visual response in an anaesthetized animal in this setup.

**Reply by niksirbi** [reply]
FYI, we have seen that you can get visual responses even with eyes closed and covered with ointment, with a bright enough LED blinking at 20Hz (in an otherwise dark magnet) in a block design. Of course, you can't do retinotopy or anything remotely specific, but the visual cortex as a whole "lights up".

**Comment by PRIME-DRE** [comment]
Contact lenses?

From: 'Chris Klink' via NHP-MRI ***@***.***>
Sent: Wednesday, November 22, 2023 12:09 PM
To: PRIME-RE/prime-re.github.io ***@***.***>
Cc: PRIME-DRE account for general things ***@***.***>; Comment ***@***.***>
Subject: [EXTERNAL] Re: [PRIME-RE/prime-re.github.io] Anesthesia & fMRI (Discussion #71)


And what do people use to keep the eyes open and hydrated when doing anaesthetized visual stimulation in the magnet? We're not after anything precise, but are looking to verify that we can measure a visual response in an anaesthetized animal in this setup.

-
Reply to this email directly, view it on GitHub<https://github.com/PRIME-RE/prime-re.github.io/discussions/71#discussioncomment-7644712>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BBSQQAMAFZMC7RYFYED5FPLYFYWTJAVCNFSM542OYVX2U5DIOJSWCZC7NNSXTOKENFZWG5LTONUW63SDN5WW2ZLOOQ5TONRUGQ3TCMQ>.
You are receiving this because you commented.Message ID: ***@***.***>
--
You received this message because you are subscribed to the Google Groups "NHP-MRI" group.
To unsubscribe from this group and stop receiving emails from it, send an email to ***@***.******@***.***>.
To view this discussion on the web visit https://groups.google.com/d/msgid/nhp-mri/PRIME-RE/prime-re.github.io/repo-discussions/71/comments/7644712%40github.com<https://groups.google.com/d/msgid/nhp-mri/PRIME-RE/prime-re.github.io/repo-discussions/71/comments/7644712%40github.com?utm_medium=email&utm_source=footer>.
CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and are confident the content is safe.

**Comment by PRIME-DRE** [comment]
Hey Kriss,
We used a custom-made irrigating-lid speculae, which was continuously infusing saline at 0.07 ml/min. This irrigation device is placed inside the eyelids.
More specifically, this speculae consisted in two loops (one for the upper lid and one for the lower lid). Both of these loops have small opening that keep irrigation constant.

***@***.***

Some refs:
PMID: 29307559
Pharmaco-Based fMRI and Neurophysiology in Non-Human Primates | SpringerLink<https://link.springer.com/protocol/10.1007/978-1-4939-6490-1_3>

Let me know if you need more info

--------------------------------------------------------------
Daniel Zaldivar (no need for labels)
A proud member from the  Scientific Community
Email: ***@***.******@***.***> or ***@***.******@***.***>
Phone: +1 301-827-5082
Mobile: +1 240-383-5433

From: 'Chris Klink' via NHP-MRI ***@***.***>
Sent: Wednesday, November 22, 2023 12:09 PM
To: PRIME-RE/prime-re.github.io ***@***.***>
Cc: PRIME-DRE account for general things ***@***.***>; Comment ***@***.***>
Subject: [EXTERNAL] Re: [PRIME-RE/prime-re.github.io] Anesthesia & fMRI (Discussion #71)


And what do people use to keep the eyes open and hydrated when doing anaesthetized visual stimulation in the magnet? We're not after anything precise, but are looking to verify that we can measure a visual response in an anaesthetized animal in this setup.

-
Reply to this email directly, view it on GitHub<https://github.com/PRIME-RE/prime-re.github.io/discussions/71#discussioncomment-7644712>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BBSQQAMAFZMC7RYFYED5FPLYFYWTJAVCNFSM542OYVX2U5DIOJSWCZC7NNSXTOKENFZWG5LTONUW63SDN5WW2ZLOOQ5TONRUGQ3TCMQ>.
You are receiving this because you commented.Message ID: ***@***.***>
--
You received this message because you are subscribed to the Google Groups "NHP-MRI" group.
To unsubscribe from this group and stop receiving emails from it, send an email to ***@***.******@***.***>.
To view this discussion on the web visit https://groups.google.com/d/msgid/nhp-mri/PRIME-RE/prime-re.github.io/repo-discussions/71/comments/7644712%40github.com<https://groups.google.com/d/msgid/nhp-mri/PRIME-RE/prime-re.github.io/repo-discussions/71/comments/7644712%40github.com?utm_medium=email&utm_source=footer>.
CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and are confident the content is safe.

**Comment by pcklink** [comment]
Thanks for all the feedback guys. Were dealing with a small Bruker 7T here so there's not a lot of space and therefore we were thinking of backprojecting onto a small inset screen. Because this is not our main project but more of a "let's see if this works" thing, we're trying to work with what we have around. 

There's a small LED projector which isn't terribly bright so I think we may want to keep the eyes open (I'd be interested in a more detailed description of what worked for you with eyes closed though, Niko, as that would simplify things a bit). Having some drip irrigation in a plastic eye speculum seems the way to aim for. We can maybe adjust something we buy (@Caspar, which ones do you use?) or create something custom like Daniel's solution, which looks good too (as far as I can see it). Do you have a more detailed image?

I don't think I will need any lenses as I'm really only trying to see the contrast between 30s of flashing checkerboards vs 30s of nothing, and I'm not too concerned with retinotopy. At most, We might check visual quadrants.

**Reply by niksirbi** [reply]
Actually, the [preprint](https://www.biorxiv.org/content/10.1101/2023.11.21.568138v1) *just* came out. To spare you searching through it:
> Visual stimulation was done via a flickering bright LED light source (a single 3W white light LED
element) placed at the back end of the magnet bore, about 100 cm away from the monkey’s
eyes, and computer-controlled via a TTL pulse. The light flicked at 20 Hz, and the stimulation
was presented in five blocks per run (30 baseline, followed by 30 sec ON - 30 sec OFF epochs
repeated x5). The eyes of the monkeys were closed and covered with ointment throughout all
experiments.

**Note** that this is in marmosets, not macaques, and the activation strength depends on the anesthetic protocol and dose

We also did some pilots (not in the paper) with awake human subjects with eyes closed. A flashing checkerboard back-projected onto a screen (following the same stimulus protocol as above) was very successful in eliciting strong visual responses. Also, subjectively speaking (as one of the subjects in the pilot 😉), if there is a substantial brightness difference between ON and OFF, one can very definitely perceive it through closed eyelids.

**Comment by pcklink** [comment]
This sounds like a nice and simple approach. I think I have an unused Arduino in a drawer somewhere that I can use to build something like this. Could be a fun little Christmas project.

**Comment by PRIME-DRE** [comment]
Hi Chris,

Here is a brief description of our anesthesia protocol for anesthetized Macaque experiments (in 7T Magnetom). 

"Two adult female macaques (5-6 kg) were used in this study. All procedures followed the guidelines of the National Institute of Health Guide for the Care and Use of Laboratory Animals and were also approved by the Institutional Animal Care and Use Committee of Zhejiang University. Following a 12-hour fasting period prior to the experiment, animals were sedated with Zoletil (2.5 mg/kg, i.m.) and given atropine (0.06 mg/kg, i.m.) to reduce mucus production. Animals were then intubated with an endotracheal tube and placed on a ventilator. The animal was placed in an MR-compatible stereotaxic with face oriented towards the front, viewing a visual monitor. 1% atropine sulfate eye drops were used to maintain mydriasis, and appropriate curvature contact lenses were placed (Danker Laboratories Inc., Sarasota, FL) for focusing on the screen. During functional scanning, anesthesia was maintained with a continuous infusion of sufentanil (2-4 µg/kg/h, i.v.; induction, 3 μg/kg) supplemented with 0.2-0.5% isoflurane. A light dosage of a paralytic (vecuronium bromide; 0.025-0.05 mg/kg/h, i.v.) was administered to reduce eye drift. Body temperature was maintained at 37-38 °C using a water blanket. Heart rate, SpO2, end-tidal CO2 and rectal temperature were continuously monitored. Anesthetic depth was assessed continuously by maintaining stable heart rate, CO2, and SpO2 levels."

Also, for eyes, the eyelids are retracted using small flexible speculae, and eyes are dilated and focused onto a tangent screen 43cm in front of the animal using gas-permeable contact lenses; I can send you contact lens info if you need it.

Anna

-----Original Messages-----
From:"'Chris Klink' via NHP-MRI" ***@***.***>
Sent Time:2023-11-23 18:45:48 (Thursday)
To: "PRIME-RE/prime-re.github.io" ***@***.***>
Cc: "PRIME-DRE account for general things" ***@***.***>, Comment ***@***.***>
Subject: Re: [PRIME-RE/prime-re.github.io] Anesthesia & fMRI (Discussion #71)






This sounds like a nice and simple approach. I think I have an unused Arduino in a drawer somewhere that I can use to build something like this. Could be a fun little Christmas project.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***>




--
You received this message because you are subscribed to the Google Groups "NHP-MRI" group.
To unsubscribe from this group and stop receiving emails from it, send an email to ***@***.***
To view this discussion on the web visit https://groups.google.com/d/msgid/nhp-mri/PRIME-RE/prime-re.github.io/repo-discussions/71/comments/7650701%40github.com.








____________________________________

Anna Wang Roe, Director & Professor

Zhejiang University Interdisciplinary Institute of Neuroscience and Technology (ZIINT)

268 KaiXuan Road, KeXueLou Room 205, Huajiachi Campus

Zhejiang University

Hangzhou, China 310029
http://www.ziint.zju.edu.cn
